src/govuk/settings/_colours-organisations.scss

```diff
@@ -70,6 +70,10 @@ $govuk-colours-organisations: (
     colour: #00ad93,
     colour-websafe: #39836e
   ),
+  "foreign-commonwealth-development-office": (
+    colour: #012169,
+    colour-websafe: #003366
```

Comment:
We (confusingly) use 'websafe' here to mean 'meets the required contrast requirements to allow it to be used for text' (which would be at least 4.5:1). This is designed to be used when brand colours would not normally meet contrast, so that a darker alternative can be specified which does.

`#012169` has a contrast ratio of 14.75:1 against white, so already meets contrast requirements just fine.

You can omit the `colour-websafe` property entirely (like has been done for `"government-equalities-office"`) and the [`govuk-organisation-colour`](https://frontend.design-system.service.gov.uk/sass-api-reference/#govuk-organisation-colour) mixin will always return `#012169` for the FCDO whether `$websafe` is true or false.

Reply:
Ah I see, thanks @36degrees ! I've removed the `colour-websafe` property now.